helm-charts/anchor-platform/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+appVersion: "1.0"
+description: Anchor Platform 2.0 Helm Chart
+maintainers:
+  - name: Philip Liu
+    email: philip.liu@stellar.org
+sources:
+  - https://github.com/stellar/java-stellar-anchor-sdk
+name: anchor-platform
+version: 0.1.0
```

Comment:
this verison number should be greater than the current existing anchor platform chart?

Reply:
I was thinking of this as a completely separate chart so the versioning would start at 0.1.0. I can rename it though and bump the version to 1.0.